### PR TITLE
Redesigned Options page for chrome. Also fixes #304

### LINKED ISSUE #304: Fix page title of extension options
Right now in Chrome the page title of the extension options page is `New Options`.  This should be updated to a more meaningful title such as `Options`.

### PR DESCRIPTION
**Pull request by prastut**
Documenting PR:
- Redesigning of home page
  - Used Bootstrap effectively to bring in symmetry.
  - Optimized the new design for all screen sizes
- Fixes isse #304

### CODE REVIEW

**Comment by irdan** [comment]
This diff was infinitely easier to review when [adding](https://github.com/blog/967-github-secrets) `?w=1` to the end of the url to ignore whitespace changes.  I have no idea why github lacks a gui switch for this.

**Comment by prastut** [comment]
@irdan I know right! I might have indented the code properly therefore the number of white-spaces diff would be high. Let me know if I can do something on my end.

File: Pages/css/options.css [hunk 1]
```diff
@@ -78,3 +78,11 @@ input {
...
   line-height: 24px;
 }
+
+.top-buffer { 
+  margin-top: 30px; 
+}
+
+h2 {
+  font-weight: bold;
+}
```
Comment (smcgregor) [inline Pages/css/options.css]: nit: needs newline at end of file.

**Comment by smcgregor** [comment]
This is in my queue to pull and test, unless @irdan beats me to it. The code looks reasonable.

**Comment by smcgregor** [comment]
The only issue I have is the nit picking inline comment I left about the newline character at the end of files. Have you squashed commits with git before? If you fix the newline, squash the commits, and update this PR I'll merge.

**Comment by prastut** [comment]
No I haven't squashed commits before? Any link to a tutorial would be
appreciable since I did try some resources but I wasn't able to understand.
:/

**Comment by smcgregor** [comment]
To understand what squashing is you only need to think of git commits as "deltas", which are incremental changes to the code. Git stores the text you add or remove since this doesn't require storing the entire file for each revision.

So a git repository is populated by a large number of deltas, and each commit is associated with a set of deltas. These commits layer on top of one another, which is what you see when you type `git log`. If you "squash" commits, then you are rewriting this commit history by combining the deltas from 2 or more commits.

I think [this should work for you](http://stackoverflow.com/questions/5189560/squash-my-last-x-commits-together-using-git/5201642#5201642), but I caution you to [copy the repository before messing with this.](https://xkcd.com/1597/) It is really easy to mess up when you are rewriting history.

**Comment by prastut** [comment]
Hahaha! I love the ample amount of sarcasm oozing from this xkcd comic. 
Commit squashed. Thanks @smcgregor for the link. :)

**Comment by smcgregor** [comment]
Merged :)

**Pull request merged**